Doc/c-api/time.rst

```diff
@@ -0,0 +1,132 @@
+.. highlight:: c
+
+PyTime C API
+============
+
+.. versionadded:: 3.13
+
+PyTime API
+----------
+
+The PyTime_t API is written to use timestamp and timeout values stored in
+various formats and to read clocks.
+
+The :c:type:`PyTime_t` type is an integer to support directly common arithmetic
+operations such as ``t1 + t2``.
+
+The PyTime_t API supports a resolution of ``1`` nanosecond. The
+:c:type:`PyTime_t` type is signed to support negative timestamps. The supported
+range is around [-292.3 years; +292.3 years]. Using the Unix epoch (January
+1st, 1970), the supported date range is around [1677-09-21; 2262-04-11].
+
+Time formats:
```

Comment:
The public API only deals with `PyTime_t`, which is always nanoseconds, right? It doesn't expose `timeval` or `timespec`.